model-optimizer/extensions/front/AttributedRollToRoll.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from extensions.ops.roll import Roll
+from mo.front.common.replacement import FrontReplacementOp
+from mo.front.tf.graph_utils import create_op_with_const_inputs
+from mo.graph.graph import Graph, rename_nodes
+
+
+class AttributedRollToRoll(FrontReplacementOp):
+    """
+    This transformation converts AttributedRoll operation (axes and shift are specified as attributes) to Roll
+    operation (Inference Engine semantic).
+    """
+    enabled = True
+    op = 'AttributedRoll'
+
+    def replace_sub_graph(self, graph: Graph, match: dict):
```

Comment:
Please, refactor the transformation as `model-optimizer/extensions/front/AttributedPadToPad.py` (use `FrontReplacementPattern` as a base class and use method `find_and_replace_pattern`).
The reason why we should avoid usage of `FrontReplacementOp` is that it is far more expensive than the approach used in `AttributedPadToPad.py`.